core/services/relay/evm/mercury/persistence_manager_test.go

```diff
@@ -23,7 +23,7 @@ func bootstrapPersistenceManager(t *testing.T, jobID int32, db *sqlx.DB) (*Persi
 	t.Helper()
 	lggr, observedLogs := logger.TestLoggerObserved(t, zapcore.DebugLevel)
 	orm := NewORM(db, lggr, pgtest.NewQConfig(true))
-	return NewPersistenceManager(lggr, orm, jobID, 2, 5*time.Millisecond, 5*time.Millisecond), observedLogs
+	return NewPersistenceManager(lggr, "mercuryserver.example", orm, jobID, 2, 5*time.Millisecond, 5*time.Millisecond), observedLogs
```

Comment:
is mercuryserver.example a valid wss url? if not, why does this work? i some validation code early in the PR to ensure wss urls. where does that come and where is it tested?

Reply:
In DB and internally the scheme is always stripped. You can provide URLs in the TOML in either form:

- wss://example.com
- example.com

Are both valid.